<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Cellphone</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="cellphone">
     <property name="toolTip">
      <string>Your 10 digit cellphone number</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="password">
     <property name="toolTip">
      <string>The MXit password for your cellphone</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="captchaLabel">
     <property name="text">
      <string>CAPTCHA</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="captchaLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Please enter the digits from the above image</string>
     </property>
     <property name="text">
      <string>Fetching captcha image</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Language</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="languageComboBox"/>
   </item>
   <item>
    <widget class="QLabel" name="countryLabel">
     <property name="text">
      <string>Country</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="countriesComboBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
